skatepark/theme.json

```diff
@@ -537,6 +537,11 @@
 					"fontStyle": "normal"
 				}
 			},
+			"core/post-author": {
+				"color": {
+					"duotone": "url(#wp-duotone-red-filter)"
+				}
+			},
```

Comment:
I added this too

Reply:
It works fine!
<img width="449" alt="Screenshot 2021-09-02 at 15 58 23" src="https://user-images.githubusercontent.com/3593343/131857060-87ae23ac-cd6b-4ba2-9e13-18945f8e158e.png">